src/actions/heap/heap.ts

```diff
@@ -136,25 +150,27 @@ export class HeapAction extends Hub.Action {
                 `Processed ${rowsProcessed} rows in ${
                   rowsProcessed / HeapAction.ROWS_PER_BATCH
                 } batch requests for envId ${envId}.`,
-              )
+              logTag)
             }
           }
         } catch (err) {
-          errors.push(err)
+          errors.push(new Error("Encountered an error onRow, error: " + getErrorMessage(err)))
```

Comment:
I want to create an InternalError here, and filter them out whent he final response is generated.